lib/auto-ssl.js

```diff
@@ -0,0 +1,44 @@
+import fs from "fs"
+import { spawnSync } from "child_process"
+
+export default function (host, email, linkDest) {
+  //where simp_le should create any keys, certs, etc
+  const sourceDest = `/etc/ssl/letsencrypt/${host}`;
+
+  //create sourceDest as necessary
+  try {
+    fs.accessSync(sourceDest);
+  } catch(e) {
+    fs.mkdirSync(sourceDest);
+  }
+
+  //try to generate cert via simp_le
+  let proc = null;
+  try {
+    proc = spawnSync("/var/simp_le/venv/bin/simp_le", [
```

Comment:
Yes, the simp_le client will detect existing certificates (and try to renew them if they're close to expiration), instead of creating new ones